ui/js/redux/selectors/navigation.js

```diff
@@ -146,3 +146,9 @@ export const selectHistoryStack = createSelector(
   _selectState,
   state => state.stack
 );
+
+// returns current page attributes (scrollY, path)
+export const selectCurrentPageAttributes = createSelector(
```

Comment:
This function returns a single entry in the history stack. I would propose a name that better reflects this, perhaps something like `selectCurrentHistoryEntry` (or use "peek" somehow if we want to keep stack terminology consistent).